static/src/js/components/TemplatePreview/__tests__/TemplatePreview.test.jsx

```diff
@@ -6,22 +6,29 @@ import {
 } from '@testing-library/react'
 import { Cookies, CookiesProvider } from 'react-cookie'
 import { MockedProvider } from '@apollo/client/testing'
+import * as router from 'react-router'
+
 import {
   MemoryRouter,
   Route,
   Routes
 } from 'react-router'
 import userEvent from '@testing-library/user-event'
+import { beforeEach } from 'vitest'
```

Comment:
I'm guessing we include describe, and other helpers i a test startup script, we should just put this there because I'm sure it'll get a fair amount of usage.

Reply:
This import is not needed. I think I added this by accident, this has been removed.